README.md

```diff
@@ -1,29 +1,34 @@
 ## BUB2
-A book uploader bot that transfers documents from public libraries such as Google Books, and Panjab Digital Library etc to Internet Archive. Built for Wikimedia Tool Labs. Check out [BUB2 on Toolforge](https://bub2.toolforge.org)!
+
+A book uploader bot that transfers documents from public libraries such as Google Books, and Punjab Digital Library etc to Internet Archive. Built for Wikimedia Tool Labs. Check out [BUB2 on Toolforge](https://bub2.toolforge.org)!
```

Comment:
Make it `Punjab (Panjab)`.

Reply:
This suggestion has not been addressed.

Reply:
In README.md...i already changed 'panjab' to 'punjab'.In original file only one paragraph is present to be changed.(making 'panjab' to 'punjab').But here it's showing same repeated paragraph without changes(panjab remains as it is).